docs/README.md

```diff
@@ -26,92 +26,183 @@
+  - [TensorFlow-Mixed-Precision](user_guide/optimization.md#tensorflow-automatic-fp16-optimization)
+- [NUMA Optimization](user_guide/optimization.md#numa-optimization)
+
+#### Scheduling and Batching
+
+Triton supports batching individual inference requests to improve compute resource utilization. This is extremely important as individual queries will not saturate GPU resources thus not leveraging the parallelism provided by GPUs to its extent. Learn more about Triton's [Batcher and Scheduler](user_guide/model_configuration.md#scheduling-and-batching).  
+- [Default Scheduler - Non-Batching](user_guide/model_configuration.md#default-scheduler)
+- [Dynamic Batcher](user_guide/model_configuration.md#dynamic-batcher)
+  - [How to Configure Dynamic Batcher](user_guide/model_configuration.md#recommended-configuration-process)
+    - [Delayed Batching](user_guide/model_configuration.md#delayed-batching)
+    - [Preferred Batch Size](user_guide/model_configuration.md#preferred-batch-sizes)
+  - [Preserving Request Ordering](user_guide/model_configuration.md#preserve-ordering)
+  - [Priority Levels](user_guide/model_configuration.md#priority-levels)
+  - [Queuing Policies](user_guide/model_configuration.md#queue-policy)
+  - [Ragged Batching](user_guide/ragged_batching.md)
+- [Sequence Batcher](user_guide/model_configuration.md#sequence-batcher)
+  - [Stateful Models](user_guide/architecture.md#stateful-models)
+  - [Control Inputs](user_guide/architecture.md#control-inputs)
+  - [Implicit State - Stateful Inference Using a Stateless Model](user_guide/architecture.md#implicit-state-management)
+  - [Sequence Scheduling Strategies](user_guide/architecture.md#scheduling-strateties)
+    - [Direct](user_guide/architecture.md#direct)
+    - [Oldest](user_guide/architecture.md#oldest)
+
+#### Rate Limiter
+Rate limiter manages the rate at which requests are scheduled on model instances by Triton. The rate limiter operates across all models loaded in Triton to allow cross-model prioritization. [Learn more](user_guide/rate_limiter.md).
+
+#### Model Warmup
+For a few of the Backends(check [Additional Resources](README.md#resources)) some or all of intialization is deffered till the first inference request is received, the benefit is resource conservation but comes with the downside of the initial requests getting processed slower than expected. Users can pre-"warm up" the model by instructing Triton to intialize the model. [Learn more](user_guide/model_configuration.md#model-warmup). 
+
+#### Inference Request/Response Cache
+Triton has a (Beta) feature which allows inference responses to get cached. [Learn More](https://github.com/triton-inference-server/server/blob/main/docs/response_cache.md).
+
+### Model Pipeline
+Building ensembles is as easy as adding an addition configuration file which outlines the specific flow of tensors from one model to another. Further changes in existing (individual)model configurations might be needed base on the use case. 
```

Comment:
Nit: Add space after parenthesis. I think it might even be easier to delete the second sentence. I think it's hard to get across that you might need to update the component models in the ensemble, and I think it should be clear from the rest of the documentation.. or maybe replace with a sentence that says that you can set the configurations for the individual models in their respective model configurations.